support/src/main/java/org/akvo/flow/support/net/ReamazeService.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2021 Stichting Akvo (Akvo Foundation)
+ *
+ * This file is part of Akvo Flow.
+ *
+ * Akvo Flow is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Akvo Flow is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Akvo Flow.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.akvo.flow.support.net
+
+import okhttp3.ResponseBody
+import retrofit2.http.Body
+import retrofit2.http.GET
+import retrofit2.http.Headers
+import retrofit2.http.POST
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface ReamazeService {
+
+    @Headers("Accept: application/json")
+    @POST("api/v1/conversations/{slug}/messages")
+    suspend fun uploadResponse(@Path("slug") slug: String, @Body messageResponse: MessageResponse): ResponseBody
+
+    @Headers("Accept: application/json")
+    @GET("api/v1/conversations/")
+    suspend fun fetchConversations(@Query("page") page: Int): ConversationsResult
```

Comment:
Does this fetch only the latest/active conversations?

Reply:
unarchived conversations are fetched but there is a paginatiion so it the first page does not contain what we want we fetch the next one